subprojects/gst-plugins-base/gst-libs/gst/gl/gstglupload.c

```diff
@@ -2255,6 +2277,11 @@ gst_gl_upload_get_input_template_caps (void)
   for (i = 0; i < G_N_ELEMENTS (upload_methods); i++) {
     GstCaps *template =
         gst_static_caps_get (upload_methods[i]->input_template_caps);
+
+    if (gst_egl_caps_has_feature (template, GST_CAPS_FEATURE_MEMORY_DMABUF)) {
+      template = gst_egl_caps_embed_modifiers (template, NULL);
```

Comment:
This behavior is not needed, I think. 
Here, you create a new gldisplay and glcontext by your self and list all modifiers in your sink template.
I think, just as the document says, we can leave this "format" field absent when we can not enum all modifiers, because you can not decide which kind of display the pipeline will use here.